### ISSUE TITLE
[ACTION] Jotform new actions

### ISSUE THREAD

**Issue by michelle0927** [PipedreamHQ/pipedream#1456]
**Is there a specific app this action is for?**
Jotform

**Please provide a link to the relevant API docs for the specific service / operation.**
https://api.jotform.com/docs/

**Create a New Form** | Add new forms with questions, properties and email settings
**Get Monthly User Usage** | Get number of form submissions received this month. Also, get number of SSL form submissions, payment form submissions and upload space used by user
**Get Form Submissions** | List of form responses
**Get User Submissions** | Get a list of all submissions for all forms on the account

**Comment by dannyroosevelt** [comment]
In addition to creating new component actions for Jotform, we also need to update the app file for the Jotform source component. Primarily, we need to abstract the URL host and API key field in the headers, in order to support distinct domains for EU vs US API keys. We're going to make the accompanying change to the app configuration as well, and we'll stage the updates to go out when this PR is ready.

**Comment by dannyroosevelt** [comment]
Specifically, we'll be using this format for the URL:
`https://{this.$auth.region}.jotform.com` (eu

**Comment by dannyroosevelt** [comment]
@michelle0927 if you don't mind, let's prioritize the updates and refactor to the source ahead of the new component actions for Jotform, since that will immediately unblock the specific user.

**Comment by dylburger** [comment]
In addition to actions support, let's also:

* Do a general refactor of the source / app file to bring it up to the current component standards
* [This user encountered a specific issue](https://pipedream.com/community/t/error-when-trying-to-add-jotform-api-key/1212/9) trying to list more than 20 forms in the source (we currently don't appear to paginate that output), so we'll need to address that specifically.

**Comment by vellames-turing** [comment]
Hey guys, can I start a review with this 3 actions or @michelle0927 will finish the last one?
Cc @dylburger @dannyroosevelt

**Comment by michelle0927** [comment]
> Hey @michelle0927. Will you finish the last action in the opened PR?

I haven't been able to get the endpoint(s) for creating forms to work properly, and @dannyroosevelt suggested I submit the other three. I've created a separate issue (https://github.com/PipedreamHQ/pipedream/issues/1927) for the Create Form action.

**Comment by vellames-turing** [comment]
> > Hey @michelle0927. Will you finish the last action in the opened PR?
> 
> I haven't been able to get the endpoint(s) for creating forms to work properly, and @dannyroosevelt suggested I submit the other three. I've created a separate issue (#1927) for the Create Form action.

I see, thanks for the update, I will start the review so
